Sming/Arch/Esp8266/Compiler/ld/common.ld

```diff
@@ -156,27 +161,30 @@ SECTIONS
 
     *libmbedtls.a:(.literal.* .text.*)
 
-    *libm.a:(.literal .text .literal.* .text.*)
+	/* C++ vtables */
+    *(.rodata._ZTV*)
 
-    *(.rodata._ZTV*) /* C++ vtables */
-    *(.irom0.literal .irom.literal .irom.text.literal .irom0.text .irom0.text.* .irom.text .irom.debug.*)
+    *(.irom0.literal .irom.literal .irom.text.literal .irom0.text .irom0.text.* .irom.text .irom.text.* .irom.debug.*)
 
     /* Generated libraries */
     *liblwip2.a:(.literal .text .literal.* .text.*)
     */clib-*.a:*(.literal .text .literal.* .text.* .stub .gnu.warning .gnu.linkonce.literal.* .gnu.linkonce.t.*.literal .gnu.linkonce.t.* .irom.debug.*)
 
+    /* Constant strings in flash (PSTRs) */
+    *(.irom0.pstr.*)
```

Comment:
This isn't actually necessary since `__fstr__` below catches it, but doesn't hurt.